docs/pages/eas-update/rollouts.mdx

```diff
@@ -2,24 +2,29 @@
 title: Rollouts
 hideTOC: true
 ---
+
 import { Terminal } from '~/ui/components/Snippet';
 
+> This feature is in Developer Preview. It's not advised to use this feature in any production environments.
+
+Rollouts incrementally deploy updates on a new branch to a specific channel. With rollouts, we can roll out updates from one branch to a percentage of end users and leave the remaining percentage of users on the current branch. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to our production environment.
 
-Rollouts allow deploying a new branch to a specific channel incrementally. With rollouts, we can roll out one branch to a percentage of end users and leave the remaining percentage of users on the current branch. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to your production environment.
+To start a rollout, we use `npx eas-cli@latest` or ensure we have version \<TODO\> or above for `eas-cli`. Then we run the following command:
 
-To initiate a rollout, use the following command:
+<Terminal cmd={['npx eas-cli@latest channel:rollout [channel]']} />
 
-<Terminal cmd={["eas channel:rollout [channel]"]} />
+An interactive guide will assist us in selecting a channel, choosing a branch for the rollout, and setting the percentage of users for the rollout. To increase or decrease the rollout amount, we run the command again and choose the `Edit` option to adjust the rollout percentage.
 
-The interactive prompts will guide you through choosing a channel, a branch to begin rolling out, and the percentage of your users to rollout to. To increase or decrease the rollout amount, run the `eas channel:rollout [channel]` command again to adjust the rollout percentage.
+### Ending a rollout
```

Comment:
```suggestion
## End a rollout
```

For consistency reasons, let's use short-form verbs for section titles instead of gerunds. Also, let's use the [second level of heading here and for all section titles on this page](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#headings).